src/Symfony/Bundle/TwigBundle/TwigDefaultEscapingStrategy.php

```diff
@@ -1,37 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Bundle\TwigBundle;
-
-/**
- * @author Fabien Potencier <fabien@symfony.com>
- */
-class TwigDefaultEscapingStrategy
```

Comment:
And now you are adding deprecations. You might want to add this in the PR header.